University-of-New-England-Australia-Discipline-of-History.csl

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>University of New England (Australia) Discipline of History Traditional Footnoting</title>
```

Comment:
Let's make this 
University of New England, Australia (note).
Accordingly, the filename should be
`university-of-new-england-australia-note.csl`
and we'd want
`http://www.zotero.org/styles/university-of-new-england-australia-note` for both the ID and the rel=self link.